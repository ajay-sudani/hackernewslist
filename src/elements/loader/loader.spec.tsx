import React from "react";
import renderer from "react-test-renderer";
import Loader from "../loader";

describe("Loader", () => {
  it("should renders", () => {
    const tree = renderer.create(<Loader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
