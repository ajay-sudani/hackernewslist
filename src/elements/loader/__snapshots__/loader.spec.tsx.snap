// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should renders 1`] = `
<div>
  <span>
    loading
  </span>
  <span />
</div>
`;
